src/TodoList.jsx

```diff
@@ -8,7 +10,7 @@ function TodoList() {
     <div>
       <ul>
         {todos.map((todo) => (
-          <li key={todo.id}>{todo.title}</li>
+          <TodoListItem todo={todo} key={todo.id} />
```

Comment:
Conventionally you want the `key` prop to be listed first. This is an identifier and not a prop that you are passing down to the child component (cannot be accessed by child). This will work but it's much easier to read with `key` listed first,  especially as your list of props grows.